README.md

```diff
@@ -1,2 +1,49 @@
-# config
+<img src="https://cdn.edge.network/assets/img/edge-logo-green.svg" width="200">
+
+# Config
+
 Simple configuration management library
+
+[![npm version](https://img.shields.io/npm/v/@edge/config)](https://www.npmjs.com/package/@edge/config) [![npm downloads](https://img.shields.io/npm/dt/@edge/config)](https://www.npmjs.com/package/@edge/config) [![license](https://img.shields.io/npm/l/@edge/config)](LICENSE.md)
+
+- [config](#config)
+  - [Usage](#usage)
+  - [License](#license)
+
+## Usage
+
+This library provides simple configuration management, allowing defaults, type parsing, and automatic `.env` loading.
+
+```typescript
+import { Config } from '@edge/config'
+
+export class GlobalConfig extends Config {
```

Comment:
Is it necessary to extend Config?

Reply:
That's a good point. If I was using `this.getEnvArray` etc then that would make sense. Removing the protected access modifiers means that the Config class could also be used for one off config grabbing from the environment!